scripts/suite_objects.py

```diff
@@ -1535,6 +1535,16 @@ def add_var_transform(self, var, compat_obj, vert_dim):
 
         # If needed, modify vertical dimension for vertical orientation flipping
         _, vdim    = find_vertical_dimension(var.get_dimensions())
+        if vdim >= 0:
+           vdim_name  = vert_dim.split(':')[-1]
+           group_vvar = self.__group.call_list.find_variable(vdim_name)
+           vname      = group_vvar.get_prop_value('local_name')
+           lindices[vdim] = '1:'+vname
+           rindices[vdim] = '1:'+vname
+           if compat_obj.has_vert_transforms:
+               rindices[vdim] = vname+':1:-1'
+           # end if
+        # end if
         vdim_name  = vert_dim.split(':')[-1]
```

Comment:
Isn't this section (1548 - 1554) a repeat of the stuff that was moved inside the `if vdim >= 0:` (line 1538)? If so, should a test have caught this?

Reply:
good catch - the only code that definitely needed to be moved inside the if block was `vdims  = vert_dim.split(':')` so a test wouldn't have caught this. that said, I did add some more testing that would've caught the original bug.